controlplane/src/db/schema.ts

```diff
@@ -461,6 +462,7 @@ export const organizationWebhooks = pgTable('organization_webhook_configs', {
   endpoint: text('endpoint'),
   key: text('key'),
   events: text('events').array(),
+  eventsMeta: json('events_meta').$type<EventsMeta>(),
```

Comment:
As discussed. Please maintain a table where we store the graph references to a webhook request. That allows us to ensure data integrity at any given moment. Your approach is great if the configuration becomes very complex or when no integrity is required.

Reply:
Done